### ISSUE TITLE
bug of reporting system. error of the reports from the enemy team, they should not be

### ISSUE THREAD

**Issue by trainer22** [ValveSoftware/Dota2-Gameplay#13475]
### Description

urgently remove the possibility of reports from the enemy team, because anyone from the enemy team, just offended by defeat, can throw a report for a good game and as a result, good players of the dota2 community go to the bottom of decency! and return the processing of reports by the patrol service instead of automatically counting all reports without processing, as is happening now - reports are thrown for no reason oligophrenes and imbeciles to good players from the opposite team and they are not even processed by the patrol but are automatically counted. this has already led to the death of the accounts of half of the really good players of the dota2 community. take urgent measures - stop the reporting system, completely disable the ability to report from the enemy team!

### Screenshots

https://virtus-img.cdnvideo.ru/images/as-is/plain/aa/aaddfd42-01ae-432c-afde-5ec98b401dc5.png@jpg

**Comment by arturkrugol** [comment]
agree, but this's useless, Valve employees don't care about us here.
